pages/common/sdcv.md

```diff
@@ -0,0 +1,37 @@
+# sdcv
+
+> StarDict console version. Command line dictionary client.
+> Dictionaries are provided separately.
+> More information: <https://manned.org/sdcv>.
+
+- Start sdcv interactively:
+
+`sdcv`
+
+- List installed dictionaries:
+
+`sdcv --list-dicts`
+
+- Display a definition from a specific dictionary:
+
+`sdcv --use-dict {{dictionary_name}} {{search_term}}`
+
+- Lookup definition with fuzzy search:
+
+`sdcv {{search_term}}`
+
+- Lookup definition with exact search:
+
+`sdcv --exact-search {{search_term}}`
+
+- Lookup definition and get json output:
```

Comment:
Perhaps this will read better?
```suggestion
- Lookup a definition and format the output as json:
```

Reply:
Shouldn't this be JSON since it is an acronym